Packs/CuckooSandbox/Integrations/integration-CuckooSandbox_README.md

```diff
@@ -25,12 +25,15 @@
 <strong>Name</strong>: a textual name for the integration instance.</li>
 <li><strong>Server URL (e.g. <a href="https://192.168.0.1/" rel="nofollow">https://192.168.0.1</a>)</strong></li>
 <li><strong>Username (Only if your Cuckoo service requires HTTP auth)</strong></li>
+<li>In order to authenticate with an API token, insert '__token' in the "Username" textbox, and the token itself in the "Password" textbox.</li>
 <li><strong>Trust any certificate (not secure)</strong></li>
 <li><strong>Use system proxy settings</strong></li>
 </ul>
 </li>
 <li>Click <strong>Test</strong> to validate the URLs, token, and connection.</li>
 </ol>
+<h2>Configuration params</h2>
+<p>NOTE: In order to authenticate with an API token, insert '__token' in the "Username" textbox, and the token itself in the "Password" textbox.</p>
```

Comment:
we agreed to put it in `CuckooSandbox_description.md`, right? if no such, we can add the "detaileddescription" field in the yml itself.
In addition, let's add there a short explanation for how to obtain the API key, if possible.

Reply:
I can't find that file, and I'll try to find information about the API key obtaining process